api/views/pois.py

```diff
@@ -0,0 +1,104 @@
+from api import app, db
+from api.models import POI, Media, Link, Story, StoryPOI
+from api.utils import create_response, InvalidUsage
+from sqlalchemy.sql.expression import func
+from flask import Blueprint, request, jsonify
+import json
+from datetime import date
+from dateutil.parser import parse
+
+mod = Blueprint('POIS', __name__)
+
+def poi_links_media_stories(poi_dict):
+    """
+    Gets additional links, media, and stories for a POI
+    """
+    poi_links = Link.query.filter(Link.poi_id == poi_dict['_id'])
+    poi_media = Media.query.filter(Media.poi_id == poi_dict['_id'])
+    poi_stories = Story.query.join(StoryPOI, Story.id == StoryPOI.story_id).filter(StoryPOI.poi_id == poi_dict['_id'])

```

Comment:
is there a way to make this line shorter (split it across multiple lines for readability)